### ISSUE TITLE
Define accessibility evaluation and improve accessibility data classes

### ISSUE THREAD

**Issue by AaroKoinsaari** [AaroKoinsaari/accessibility-map-android#2]
Need to define how to calculate the accessibility "score" (red, yellow, green, grey) for each place. For example, if the place has an elevator but not accessible entry, is it yellow or orange or what?

Also, this task could include improving some of the models and enum classes and adding or editing their parameters if needed.
